src/main/java/org/influxdb/InfluxDB.java

```diff
@@ -168,6 +168,22 @@ public void write(final String database, final String retentionPolicy,
                     final ConsistencyLevel consistency, final List<String> records);
 
   /**
+   * Write a set of Points to the influxdb database with the string records through UDP.
+   *
+   * @param udpPort
+   * @param records
+   */
+  public void write(final int udpPort, final String records);
```

Comment:
@majst01  Can you change README.md after we complete both sync and async UDP support? It will be more better to express how to use it?  WDYT?